logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -84,24 +86,31 @@ actual class CallManagerImpl(
     }
 
     private fun startHandleAsync(): Deferred<Handle> = scope.async(start = CoroutineStart.LAZY) {
-        val selfUserId = userId.await().value
+        val selfUserId = userId.await()
         val selfClientId = clientId.await().value
 
         val waitInitializationJob = Job()
 
         val handle = calling.wcall_create(
-            userId = selfUserId,
+            userId = federatedIdMapper.parseToFederatedId(selfUserId),
```

Comment:
Seems like we are only using `selfUserId` to be mapped, maybe we can move the `federatedIdMapper.parseToFederatedId(..` directly to line 89,, such as:

`val selfUserId = federatedIdMapper.parseToFederatedId(userId.await())`

what do you think?